app/tests/unit/specs/components/staking/TableValidators.spec.js

```diff
@@ -42,7 +42,10 @@ describe(`TableValidators`, () => {
     }
 
     wrapper = shallowMount(TableValidators, {
-      propsData: { validators },
+      propsData: {
+        validators,
+        showMobileSorting: true,
```

Comment:
this is always set to true and therefore it is always visible, also on Desktop. Shouldn't it depend on the screen size?